.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,110 @@
+name: "🐛 Bug Report"
+description: "Submit a bug report to help us improve"
+title: "🐛 Bug Report: "
+labels: [bug]
```

Comment:
Could you change the labels to reflect the same "bug" label used in the repo? Not sure what the exact syntax would have to be for the yaml file, but this should be --> `🐛  Opportunity: Bug`

Reply:
If you have labels created that way then we can mention the same within brackets and it should work. Applying changes as suggested.